contracts/FlowCallbackScheduler.cdc

```diff
@@ -326,7 +326,7 @@ access(all) contract FlowCallbackScheduler {
                 executionEffort: executionEffort
             )
 
-            if estimate.error != nil {
+            if estimate.error != nil && estimate.timestamp == nil {
```

Comment:
I think this will bite someone in the ass down the line, but I'm fine with it if you think it's ok. I would just at least add a comment explaining why we do this here.

Reply:
okay I'll add that